### DISCUSSION TITLE
Shared volumes for ProjectResource

### DISCUSSION THREAD

**Discussion by eSPiYa** [microsoft/aspire#4336] [Q&A] [answered]
I'm sorry if this was already asked, but I already tried for almost 2 hours to find how to mount a shared volume to my .NET services, preferably a named ones like in Docker(Compose) that can be deleted immediately during development, but I can't find an answer. I just discovered this .NET Aspire last week but tried it just for today.

I'm checking out if our application can be converted to Aspire right away. Its main requirement right now is able to write files to a shared volume between instances.

Also, how to make sure that this shared volume can be translated to Kubernetes later?

**Comment by davidfowl** [comment]
There's no support right now for adding volumes to project resources (which get turned into containers at deployment time). I'll turn this in to an issue (feature request).

**Reply by davidfowl** [reply, marked as answer]
https://github.com/dotnet/aspire/issues/4359
